### PR TITLE
Add option to run on all integration test cases

### PR DESCRIPTION
**Pull request by vaikuntht**
Changes to run integration tests in multi-node all together, now they can be run with:

`qserv-test-integration -M`

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -11,10 +11,10 @@ qserv-check-integration.py --case=01 --load --mode=qserv
 # to run integration tests for one case, only for mysql
 qserv-check-integration.py --case=01 --load --mode=mysql
 
-# to run multi-node, use -M
-qserv-check-integration.py --case=01 --load -M --mode=qserv
+# to run multi-node tests for one case, use -M
+qserv-check-integration.py --case=01 --load -M
```
Comment (jbecla) [inline README.txt]: -M still needed here?

File: python/lsst/qserv/tests/unittest/testIntegration.py [hunk 2]
```diff
@@ -48,6 +48,8 @@
 # -----------------------
 class TestIntegration(unittest.TestCase):
 
+    runMulti = None
```
Comment (jbecla) [inline python/lsst/qserv/tests/unittest/testIntegration.py]: I think False would be better here (I traced it to the code in dbLoader where we have "if multi_node:")

Comment (vaikuntht) [inline python/lsst/qserv/tests/unittest/testIntegration.py]: Changed.

File: README.txt (hunk 1 above)
Comment (vaikuntht) [inline README.txt]: `qserv-test-integration.py` and `qserv-check-integration.py` are separate scripts, with the latter allowing a lot more control over individual options. I left the specific option to run in multi-node setup there for that purpose, but if we want to do it "automatically" for check-integration too by reading `node_type` in the config files I can do that.

**Pull request merged**